wasmcloud_host/README.md

```diff
@@ -1,19 +1,172 @@
-# WasmcloudHost
+# `wasmcloud_host`
 
-To start your Phoenix server:
+## Prerequisites
 
-  * Install dependencies with `mix deps.get`
-  * Install Node.js dependencies with `npm install` inside the `assets` directory
-  * Start Phoenix endpoint with `mix phx.server`
+1. Install the latest versions of Rust and Go
 
-Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
+1. Since the build of `wasmcloud-otp` will fail on version `16.xx.yy` of NodeJs, you should either install, or switch to, version `14.xx.yy` of NodeJS.
 
-Ready to run in production? Please [check our deployment guides](https://hexdocs.pm/phoenix/deployment.html).
+    Typically, a NodeJS version management tool such as `nvm` or `fnm` is used:
 
-## Learn more
+    E.G. To install Fast Node Manager (`fnm`):
 
-  * Official website: https://www.phoenixframework.org/
-  * Guides: https://hexdocs.pm/phoenix/overview.html
-  * Docs: https://hexdocs.pm/phoenix
-  * Forum: https://elixirforum.com/c/phoenix-forum
-  * Source: https://github.com/phoenixframework/phoenix
+    ```console
+    $ cargo install fvm
+    ```
+
+    Install the latest version of NodeJS version 14
```

Comment:
If you're teaching people how to install fvm then you want `eval "$(fvm env)"` in there somewhere too. You probably just want to link out to fvm and nvm's documentation sites, and say `fnm use 14` or `nvm use 14`.

Reply:
*fnm